### DISCUSSION TITLE
How do I find the fema drop?

### DISCUSSION THREAD

**Discussion by CalebFenton** [TEHE-Studios/ExpandedHelicopterEvents#124] [Q&A] [answered]
The drop icon pops up indicating a direction and then disappears after a few seconds. I've had so many drops and have never been able to find them. 

Do they just disappear?

One time I had a drop and had to leave the game and came back and the icon was gone and the event had been cancelled or something. Is that drop gone?

Do they make some kind of noise?

**Comment by Chuckleberry-Finn** [comment, marked as answer]
The marker disappearing is to simulate it not being visible anymore - however the direction should not change. The drop can be within a range of 200ish tiles from the event.

The drops at that point are permanent, but can be towed to be moved.
